src/ScreenStages/GameScreen.cpp

```diff
@@ -87,6 +88,7 @@ void CGameScreen::Init()
 
     g_ScreenEffectManager.UseSunrise();
     SmoothScreenAction = 0;
+    g_GumpManager.Add(new CGumpToolbar(300, 600));
```

Comment:
I think it should have an option to close/open this gump

Reply:
ok cool I dont know what you what for this I a next pr I will add an option